xfuser/model_executor/layers/attention_processor.py

```diff
@@ -121,7 +121,7 @@ def __init__(
         assert (to_k.bias is None) == (to_v.bias is None)
         assert to_k.weight.shape == to_v.weight.shape
 
-        in_size, out_size = to_k.in_features, to_k.out_features
+        '''in_size, out_size = to_k.in_features, to_k.out_features
```

Comment:
use # for comments or release these lines.

Reply:
fixed.